api/internal/core/entity/entity.go

```diff
@@ -83,6 +83,7 @@ type Route struct {
 	ServiceProtocol string                 `json:"service_protocol,omitempty"`
 	Labels          map[string]string      `json:"labels,omitempty"`
 	EnableWebsocket bool                   `json:"enable_websocket,omitempty"`
+	Status          uint8                  `json:"status"`
```

Comment:
@liuxiran @nic-chen 
I guess this line need using a pointer to `Status` type, and should be changed to
```suggestion
	Status          *Status                  `json:"status"`
```

With `uint8`, we can not distinguish whether the `status` field is unset or set to `0`.

Reply:
sure, we can optimize this place. but they are not in the same bag and cannot be used directly in this way.

Reply:
> sure, we can optimize this place. but they are not in the same bag and cannot be used directly in this way.

@nic-chen thanks for getting back to me. Shall we also need to use `*uint8` here instead of `uint8`? While honestly, the passed CI makes me rather confused, am I missing anything that could explicitly tell us the `status` param is **unset** or **set to 0** ? Any feedback is much appreciated.

Reply:
@imjoey  It's because we process the raw request body and set default value when there is no `status`

Reply:
@nic-chen Got it and it's ok for me. Thanks for the explanation.